src/lib/containers/access_requirement_list/AcceptedRequirements.tsx

```diff
@@ -183,7 +185,23 @@ export default function AcceptedRequirements({
           )}
         </div>
       </div>
-      {token && showButton && (
+      { // Show request access button for ManagedACTAccessRequirement component without logged in
+        isRequestAccessButton && showButton && (
+        <div className={`button-container ${isApproved ? `hide` : `default`}`}>
+          <div className="accept-button-container">
+            <button className="accept-button" onClick={onAcceptClicked}>
```

Comment:
If anonymous (no token), I'd expect onAcceptClicked to pop up a message instead sending to Synapse.org (or opening up your new wizard).

Reply:
Way to avoid duplicating the UI code?  (isRequestAccessButton || token) && showButton && ...

Reply:
I am unclear whether it's okay to keep the current workflow to redirect to Synapse.org and over there, user will be asked to first sign in. If  the workflow needs to stay in portals, then the changes should probably go with the new feature branch.